### PR TITLE
Added Summer of Bitcoin

### PR DESCRIPTION
**Pull request by TsarPrince**

### CODE REVIEW

**Comment by saintmalik** [comment]
Thanks for adding this 😊, seems you missed out something, sending Pull Request from your main branch directly, it can cause conflicts most times, but it wont stop your PR from getting merge👍.

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@
 | ⭐ [Linux Kernel Mentorship Program](https://wiki.linuxfoundation.org/lkmp)            | Yes  | [Mar - May](https://wiki.linuxfoundation.org/lkmp/lkmp_schedule)  | mid-Jan             | late-Feb   | Mentorship Program  |  Certificate  |
 | ⭐ [Hyperledger Mentorship Program](https://wiki.hyperledger.org/display/INTERN/How+to+Apply)             | Yes  | [Jun - Aug(Full-Time) Jun - Nov(Part-time)](https://wiki.hyperledger.org/display/INTERN#HyperledgerMentorshipProgram-SummerFT(Full-Time)MenteeSchedule)  | Jan-28                | Mar-10       | Mentorship Program   | Certificate  |
 | ⭐ [Outreachy](https://www.gnome.org/outreachy/)                                  | Yes  | [May - Aug](https://www.outreachy.org/apply/project-selection/)  | late Jan            | late Feb | Internship  | Certificate  |
+| [Summer of Bitcoin](https://www.summerofbitcoin.org/) | Yes (3000 USD) | [Jan - Aug](https://www.summerofbitcoin.org/how-it-works) | January 1 | March 15 | Summer Internship Program | Internship, Job opportunity (if performed exceptionally well)
```
Comment (ayush-sleeping) [inline README.md:98]: looking Perfect .

**Comment by ayush-sleeping** [comment]
Well done @TsarPrince

File: README.md (hunk 1 above)
Comment (TsarPrince) [reply README.md:98]: @saintmalik  thanks for the suggestion, will keep this in mind😊

**Comment by TsarPrince** [comment]
@saintmalik thanks for the suggestion, will keep this in mind😊

**Pull request merged**